resources/functionMap_php80delta.php

```diff
@@ -38,6 +38,7 @@
 		'date_time_set' => ['DateTime', 'object'=>'DateTime', 'hour'=>'int', 'minute'=>'int', 'second='=>'int', 'microseconds='=>'int'],
 		'date_timestamp_set' => ['DateTime', 'object'=>'DateTime', 'unixtimestamp'=>'int'],
 		'date_timezone_set' => ['DateTime', 'object'=>'DateTime', 'timezone'=>'DateTimeZone'],
+		'define\'1' => ['bool', 'constant_name'=>'string', 'value'=>'mixed'],
```

Comment:
Adding new variant isn't going to solve this. Please revert all the changed code and add this here without the number. That will overwrite the definiton from functionMap.php

Alternatively (since PHP 8 technically still has that argument, it just doesn't do anything), we could implement this as a brand new rule without touching the functionMap at all. But that would require new code.

Reply:
I was wondering what approach to take. I think it probably needs a new rule since using the third argument triggers a warning not an error

Reply:
I'm also for a new rule so that we can have a custom message too.